src/model.ts

```diff
@@ -97,6 +175,105 @@ export function getDefaultEnumValues(prop: Property, schema: Schema, opt: QueryC
     case Property.AGGREGATE:
       return opt.aggregates;
 
+    case Property.AXIS_AXISCOLOR:
```

Comment:
This makes me wonder if it's a mistake to remove that old `+ 's'` code.  But nvm, you already typed all these, so let's stick with lol.